pkgs/development/tools/flamegraph/default.nix

```
@@ -1,13 +1,13 @@
 { stdenv, fetchFromGitHub, perl }:
 
 stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation rec {
```
(in order to be able to set `repo = pname` further down)